R/execution.r

```diff
@@ -237,6 +237,11 @@ execute = function(request) {
             if (!plot_builds_upon(last_recorded_plot, plotobj)) {
                 send_plot(last_recorded_plot)
             }
+            # need to be set here to capture the size and have it available
+            # when the plot is sent
+            # 7 is the default in repr...
+            attr(plotobj, '.irkernel_width') <- getOption("repr.plot.width", 7)
+            attr(plotobj, '.irkernel_height') <- getOption("repr.plot.height", 7)
```

Comment:
why wait? i think you can expect IRkernel dev versions to not work without the newest repr